packages/e2e/test/k6/scenarios/idle-clients.js

```diff
@@ -0,0 +1,220 @@
+  activeAddrCount: 10,
+  /** Use only addresses with a transaction history up to this value */
+  maxTxHistory: 100
+};
+
+/** Repetitive endpoints */
+const TIP_URL = 'network-info/ledger-tip';
+
+export const options = {
+  ext: {
+    loadimpact: {
+      apm: [],
+      distribution: { 'amazon:de:frankfurt': { loadZone: 'amazon:de:frankfurt', percent: 100 } }
+    }
+  },
+  scenarios: {
+    Scenario_1: {
+      exec: 'scenario_1',
+      executor: 'ramping-vus',
+      gracefulRampDown: '0s',
+      gracefulStop: '0s',
+      stages: [
+        { duration: RAMP_UP_DURATION, target: MAX_VU },
+        { duration: STEADY_STATE_DURATION, target: MAX_VU }
+      ]
+    }
+  },
+  thresholds: { http_req_duration: ['p(95)<200'], http_req_failed: ['rate<0.02'] }
+};
+
+/** equivalent to lodash.chunk */
+const chunkArray = (array, chunkSize) => {
+  const arrayCopy = [...array];
+  const chunked = [];
+  while (arrayCopy.length > 0) {
+    chunked.push(arrayCopy.splice(0, chunkSize));
+  }
+  return chunked;
+};
+
+/** Util functions for sending the http post requests to cardano-sdk services */
+const cardanoHttpPost = (url, body = {}) => {
+  const opts = { headers: { 'content-type': 'application/json' } };
+  return http.post(`${PROVIDER_SERVER_URL}/${url}`, JSON.stringify(body), opts);
+};
+
+const utxosByAddresses = (addresses) => {
+  const addressChunks = chunkArray(addresses, 25);
+  for (const chunk of addressChunks) {
+    cardanoHttpPost('utxo/utxo-by-addresses', { addresses: chunk });
+  }
+};
+
+/**
+ *
+ * @param addresses Bech32 cardano addresses: `Cardano.Address[]`
+ * @param blockHeightOfLastTx query transactions done starting with this block height.
+ */
+const txsByAddress = (addresses, blockHeightOfLastTx) => {
+  const pageSize = 25;
+  const addressChunks = chunkArray(addresses, pageSize);
+  for (const chunk of addressChunks) {
+    let startAt = 0;
+    let txCount = 0;
+
+    do {
+      const resp = cardanoHttpPost('chain-history/txs/by-addresses', {
+        addresses: chunk,
+        blockRange: { lowerBound: blockHeightOfLastTx },
+        pagination: { limit: pageSize, startAt }
+      });
+
+      if (resp.status !== 200) {
+        // No point in trying to get the other pages.
+        // Should we log this? it will show up as if the restoration was quicker since this wallet did not fetch all the pages
+        break;
+      }
+
+      const { pageResults } = JSON.parse(resp.body);
+      startAt += pageSize;
+      txCount = pageResults.length;
+    } while (txCount === pageSize);
+  }
+};
+
+/**
+ * Grab the wallets json file to be used by the scenario.
+ * Group the addresses per wallet (single address or HD wallets).
+ */
+export function setup() {
+  console.log(
+    `Ramp-up: ${RAMP_UP_DURATION}; Sustain: ${STEADY_STATE_DURATION}; Poll: ${POLL_INTERVAL}s; Block height change every: ${NUM_TIP_QUERIES}`
+  );
+  const res = http.get(WALLET_ADDRESSES_URL);
```

Comment:
looks like this call will be included in our statistic and there is no way around this so far
https://github.com/grafana/k6/issues/1321

Reply:
I will add an inline comment

Reply:
done here: [db0bad8](https://github.com/input-output-hk/cardano-js-sdk/pull/808/commits/db0bad8c05b490403264bb49e5a9d101b66067ec)